manifests/plugin/rh_cloud.pp

```diff
@@ -1,17 +1,5 @@
 # @summary Installs rh_cloud plugin
-#
-# @param enable_iop_advisor_engine
-#   Enable iop-advisor-engine integration
-#
-class foreman::plugin::rh_cloud (
-  Boolean $enable_iop_advisor_engine = false,
-) {
+class foreman::plugin::rh_cloud {
   foreman::plugin { 'rh_cloud':
-    config => epp('foreman/rh_cloud.yaml.epp', { 'enable_iop_advisor_engine' => $enable_iop_advisor_engine }),
-  }
-
-  class { 'iop_advisor_engine':
```

Comment:
I suppose we should also remove the leftovers of the `iop_advisor_engine` class too here?
Another option is to deal with it in the migration path.

Reply:
That happens over here -- https://github.com/theforeman/puppet-iop/blob/master/manifests/init.pp#L61